react-app/src/NavBar.js

```diff
@@ -0,0 +1,60 @@
+import {
+  StyleSheet,
+  Text,
+  View,
+  Pressable,
+  Dimensions,
+  Alert,
+} from 'react-native'
+import React from 'react'
+
+export default function NavBar() {
+  return (
+    <View style={styles.container}>
+      <Pressable
+        onPress={() => Alert.alert('This is a recall function.')}
+        style={styles.button}
+      >
+        <View style={styles.bar}></View>
+        <View style={styles.bar}></View>
+        <View style={styles.bar}></View>
+      </Pressable>
+
+      <Text style={styles.text}>Relief in Seat</Text>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    // flex: 1,
```

Comment:
Do we need this?

Reply:
Yes, as far as I know, we need this to style our app